### ISSUE TITLE
Message from GelPell is added everytime I answer to any email in my inbox or compose a new email

### ISSUE THREAD

**Issue by sfoshaddck** [tutao/tutanota#3768]
**Describe the bug**
Everytime I answer any email or compose a new email I get garbage in the body. I don't remember ever being in contact with this company whatever they do so I don't want it in my future emails ever again.

**To Reproduce**
1. Open Tutanota's Desktop client.
2. Click on New Message.
3. At the new message window in its body you'll see a message from Kristina Edvinsson to her colleague's.

**Expected behavior**
There should be a blank new body to type in.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Mageia 8 for x86_64 architecture.
 - Desktop Environment: Cinnamon 4.8.3.
 - Version: v3.89.25

**Comment by sfoshaddck** [comment]
I made a Reddit post about the problem here as well: https://www.reddit.com/r/tutanota/comments/rzwjqf/message_from_gelpell_is_added_everytime_i_answer/

**Comment by Umeaboy** [comment]
Problem solved temporarily by checking Settings ->E-mail ->E-mail signature and changing it from Custom to None.
It still doesn't explain how the text ended up in the body, but closing as FIXED.

Should anyone feel that they still experience this issue, please feel free to reopen with new information.

**Comment by Umeaboy** [comment]
Seems like I can't get it closed. Anyone mind closing for me?

**Comment by charlag** [comment]
@Umeaboy you didn't open it, is it also your account?

**Comment by Umeaboy** [comment]
> @Umeaboy you didn't open it, is it also your account?

Yeah, it is. Sorry, I forgot that I opened it with that other account. I'll relogin and close it with that.

**Comment by sfoshaddck** [comment]
Closing now as FIXED.
